__tests__/git.test.ts

```diff
@@ -227,8 +224,81 @@ index ec952c3..4db9502 100644
   ]);
 
   // act
-  const actualRanges = git.parseChangedLines(diff, worktreeRoot);
+  const actualRanges = git.parseChangedLines(TEST_DIFF, worktreeRoot);
 
   // assert
   expect(actualRanges).toEqual(expectedRanges);
 });
+
+test(`parseDiffHunks correctly parses diff hunks from two files`, async () => {
+  // arrange
+  const expectedHunks = [
+    {
+      leftRange: undefined,
+      leftFile: '__tests__/git.test.ts',
+      rightRange: {start: 165, end: 165 + 29},
```

Comment:
29 is the number of lines added to the revision but if you store it like that, won't it get evaluated? I see that is intended also [here](https://github.com/SpoonLabs/sorald-buildbreaker/blob/54c6c972dcaacf9098e86b57c3ccabe1e289780f/src/ranges.ts#L6). I am not sure if storing this would be useful for you so I thought I'd point it out.

Reply:
The end position is `start + numLines` (actually, it's `start + numLines - 1`, but #59 will fix that). A range must store the start and end. I could have written `184` instead, but it's less clear where that magic number comes from as it doesn't appear in the diff.